### PR TITLE
Update get-started-partner-integration.md

### PR DESCRIPTION
**Pull request by EfiKliger**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/get-started-partner-integration.md [hunk 1]
```diff
@@ -25,21 +25,23 @@ ms.topic: conceptual
...
 
 To become a Defender for Endpoint solution partner, you'll need to follow and complete the following steps.
 
-## Step 1: Subscribe to a Microsoft Defender for Endpoint Developer license
+## Step 1: Subscribe to a Microsoft Defender for Endpoint license
 
-Subscribe to the [Microsoft Defender for Endpoint Developer license](https://winatpregistration-prd.trafficmanager.net/Developer/UserAgreement?Length=9). Subscribing allows you to use a Microsoft Defender for Endpoint tenant with up to 10 devices to developing solutions that integrate with Microsoft Defender for Endpoint.
+Want to experience Defender for Endpoint? [Sign up for a free trial.](https://signup.microsoft.com/create-account/signup?products=7f379fee-c4f9-4278-b0a1-e4c8c2fcdf7e&ru=https://aka.ms/MDEp2OpenTrial?ocid=docs-wdatp-exposedapis-abovefoldlink). Subscribing allows you to use a Microsoft Defender for Endpoint tenant with up to 3 devices to developing solutions that integrate with Microsoft Defender for Endpoint.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/get-started-partner-integration.md]:
```suggestion
Want to experience Defender for Endpoint? [Sign up for a free trial.](https://signup.microsoft.com/create-account/signup?products=7f379fee-c4f9-4278-b0a1-e4c8c2fcdf7e&ru=https://aka.ms/MDEp2OpenTrial?ocid=docs-wdatp-exposedapis-abovefoldlink). Subscribing allows you to use a Microsoft Defender for Endpoint tenant with up to three devices to developing solutions that integrate with Microsoft Defender for Endpoint.
```

File: microsoft-365/security/defender-endpoint/get-started-partner-integration.md [hunk 2]
```diff
@@ -25,21 +25,23 @@ ms.topic: conceptual
...
 
 ## Step 2: Fulfill the solution validation and certification requirements
 
 The best way for technology partners to certify that their integration works is to have a joint customer approve the suggested integration design (the customer can use the **Recommend a partner** option in the [Partner Application page](https://securitycenter.microsoft.com/interoperability/partners) in the Microsoft Defender Security Center) and have it tested and demoed to the Microsoft Defender for Endpoint team.
 
 Once the Microsoft Defender for Endpoint team has reviewed and approves the integration, we will direct you to be included as a partner at the Microsoft Intelligent Security Association.
 
-## Step 3: Get listed in the Microsoft Defender for Endpoint partner application portal
+## Step 3: Become a MISA member 
+Managed security service providers (MSSP) and independent software vendors (ISV) can be nominated to MISA [here](https://www.microsoft.com/en-us/security/business/intelligent-security-association)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/get-started-partner-integration.md]:
```suggestion
Managed security service providers (MSSP) and independent software vendors (ISV) can be nominated to the Microsoft Intelligent Security Association (MISA) [here](https://www.microsoft.com/security/business/intelligent-security-association).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @EfiKliger, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**